staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2758,6 +2761,19 @@ const (
 	Tap DomainAttachmentType = "tap"
 )
 
+type InterfaceBindingMigration struct {
+	// Method defines a pre-defined migration methodology
+	// version: 1alphav1
+	Method MigrationMethod `json:"method,omitempty"`
+}
+
+type MigrationMethod string
+
+const (
+	// LinkRefresh method will invoke link down -> link up interface to force the guest to request new IP address and routes from DHCP
```

Comment:
I would not use "force" as we cannot really do that.
It is more like "give a chance to the guest ...".